base_rest/__manifest__.py

```diff
@@ -20,8 +20,7 @@
     "assets": {
         "web.assets_frontend": [
             "base_rest/static/src/scss/base_rest.scss",
-            "base_rest/static/src/js/swagger_ui.js",
```

Comment:
No we should keep  js  files required for the swagger UI here. These are loaded into the dedicated view.